### PR TITLE
[201911] [Innovium] Update Cameo & Wistron Drivers

### LINKED ISSUE #8068: failed to build "target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb" and "target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb" on branch 201911 on 2021/7/6
#### Steps to reproduce the issue:
1. git clone  https://github.com/Azure/sonic-buildimage.git
2. git checkout 201911 origin/201911
3. make init
4. reuse the stretch docker images by modifying Makefile and Makefile.work
5. make configure PLATFORM=innovium
6. make target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb
7. make target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb

#### Describe the results you received:
[ FAIL LOG START ] [ target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb ]
/bin/bash: line 9: pushd: platform/innovium/sonic-platform-modules-cameo: No such file or directory
[  FAIL LOG END  ] [ target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb ]
slave.mk:360: recipe for target 'target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb' failed
make: *** [target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb] Error 1
make[1]: *** [Makefile.work:202: target/sonic-innovium.bin] Error 2
make[1]: Leaving directory '/home2/jersey/work/nicholas/SilverstoneX_SONiC/merge_build/sonic-buildimage'
make: *** [Makefile:8: target/sonic-innovium.bin] Error 2

[ FAIL LOG START ] [ target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb ]
/sonic/platform/innovium/sonic-platform-modules-wistron /sonic
tail: cannot open 'debian/changelog' for reading: No such file or directory
dpkg-buildpackage: error: tail of debian/changelog gave error exit status 1
[  FAIL LOG END  ] [ target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb ]
slave.mk:360: recipe for target 'target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb' failed
make: *** [target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb] Error 1
make[1]: *** [Makefile.work:202: target/sonic-innovium.bin] Error 2
make[1]: Leaving directory '/home2/jersey/work/nicholas/SilverstoneX_SONiC/merge_build/sonic-buildimage'
make: *** [Makefile:8: target/sonic-innovium.bin] Error 2

#### Describe the results you expected:
build  target/debs/stretch/sonic-platform-cameo-esc601-32q_1.0.0_amd64.deb successfully 
build  target/debs/stretch/sonic-platform-wistron-sw-to3200k_1.1_amd64.deb successfully

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

### PR DESCRIPTION
**Pull request by shihjeff**
#### Why I did it

Fix  #8068 

Update Innovium configs on Cameo and Wistron platforms

#### How I did it

NA

#### How to verify it

Sonic PTF

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

#### Description for the changelog

Update Innovium configs + Add new platforms supporting Innovium chips

### CODE REVIEW

**Comment by lguohan** [comment]
please fix lgtm alerts.

**Comment by shihjeff** [comment]
> please fix lgtm alerts.

@lguohan All LGTM alerts fixed (except the wrong # arguments false alarms).  Please merge.

**Review by jleveque — CHANGES_REQUESTED** [review]
You are attempting to check in a shared library binary (device/wistron/x86_64-wistron_sw_to3200k-r0/lib_ivm_serdes_pltfm.so). Rather than checking in the binary, please check in the source and build the shared library when building your platform modules.

**Comment by shihjeff** [comment]
> You are attempting to check in a shared library binary (device/wistron/x86_64-wistron_sw_to3200k-r0/lib_ivm_serdes_pltfm.so). Rather than checking in the binary, please check in the source and build the shared library when building your platform modules.

@jleveque - unfortunately, this code currently is proprietary and can't be released at this time.  This is for managing the serdes settings on this platform only.

File: platform/innovium/sonic-platform-modules-cameo/debian/cameo_platform_version.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+# driver version
+
+PLATFROM=$1
```
Comment (jleveque) [inline platform/innovium/sonic-platform-modules-cameo/debian/cameo_platform_version.sh:4]: Typo. Should be "PLATFORM"

File: platform/innovium/sonic-platform-modules-cameo/debian/cameo_platform_version.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+PLATFROM_PATH=$2
```
Comment (jleveque) [inline platform/innovium/sonic-platform-modules-cameo/debian/cameo_platform_version.sh:5]: Typo. Should be "PLATFORM_PATH"

**Review by jleveque — CHANGES_REQUESTED** [review]

**Comment by jleveque** [comment]
I noticed there are also `.pyc` files (compiled Python files), which are binary. These should be removed, as well.

**Comment by shihjeff** [comment]
@jleveque - Changes to remove the *.pyc files are done.  Converted to source code.  Appreciate your time.  Thanks!

CC @Hanly-Cameo

**Review by lguohan — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Pull request merged**